lib/view/page/create_idea_page.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import 'package:idea_tracker/view/widget/state_management/base_view.dart';
+import 'package:idea_tracker/controller/page/create_idea_controller.dart';
+
+class CreateIdeaPage extends StatelessWidget {
+  final String title;
+  final _formKey = GlobalKey<FormState>();
+  final _focusNode = FocusNode();
+  void _onFieldSubmitted(String value) {
+    _focusNode.requestFocus();
+  }
+
+  CreateIdeaPage({this.title});
+
+  @override
+  Widget build(BuildContext context) {
+    return BaseView<CreateIdeaController>(
+      builder: (context, controller, child) {
+        return Scaffold(
+          appBar: AppBar(
+            centerTitle: true,
+            title: Text(title),
+            leading: IconButton(
+              icon: Icon(Icons.arrow_back),
+              onPressed: () {
+                Navigator.of(context).pop();
+              },
+            ),
```

Comment:
A back button will be added automatically after navigating to this page. This code is not needed.